app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -84,26 +84,13 @@ def edit
       end
 
       def after_create
-        if @collection.is_a?(ActiveFedora::Base)
-          form
-          set_default_permissions
-          # if we are creating the new collection as a subcollection (via the nested collections controller),
-          # we pass the parent_id through a hidden field in the form and link the two after the create.
-          link_parent_collection(params[:parent_id]) unless params[:parent_id].nil?
-        end
-        respond_to do |format|
-          Hyrax::SolrService.commit
-          format.html { redirect_to edit_dashboard_collection_path(@collection), notice: t('hyrax.dashboard.my.action.collection_create_success') }
-          format.json { render json: @collection, status: :created, location: dashboard_collection_path(@collection) }
-        end
+        Deprecation.warn("Method `#after_create` will be removed in the next major release.")
```

Comment:
Is it possible to explicitly name the release?  I think it would be most useful to anyone who gets this warning to know in which version it will be removed.
```suggestion
        Deprecation.warn("Method `#after_create` will be removed in Hyrax 4.0.")
```

Reply:
DONE